app/views/state_file/questions/md_review/edit.html.erb

```diff
@@ -1,4 +1,13 @@
 <% content_for :page_title, t("state_file.questions.shared.abstract_review_header.title") %>
+
+<% content_for :additional_household_box_content do %>
+  <div class="spacing-below-5">
+    <h3 class="text--body text--bold spacing-below-5"><%=t(".your_address", filing_year: current_tax_year) %></h3>
```

Comment:
🤔 what does this have to be with the SMS work?

Reply:
tried to split out just the sms terms but some commits made it through, so I'm closing this pr and merging this code to  https://github.com/codeforamerica/vita-min/pull/5091